src/Infrastructure/MembershipApplicationTrackingInfo.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class MembershipApplicationTrackingInfo {
+
+	private $totalImpressionCount;
+	private $singleBannerImpressionCount;
+	private $confirmationPage;
+	private $confirmationPageCampaign;
```

Comment:
Could you also modify the value objected added by this PR to reflect how you think it should be? (You can do that without doing the other thing first.)